### DISCUSSION TITLE
Trivy scan failed (failed to download vulnerability DB: TOOMANYREQUESTS)

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [goharbor/harbor#21189] [Q&A]
Hi,

Image scanning failed with the TOOMANYREQUESTS error.
How can I solve this error?
Harbor version is 2.5.0.

```
init error: DB error: failed to download vulnerability DB: OCI artifact error: OCI artifact error: OCI repository error: GET https://ghcr.io/v2/aquasecurity/trivy-db/manifests/2: TOOMANYREQUESTS: retry-after: 905.841µs, allowed: 44000/minute\n
```

**Comment by MinerYang** [comment]
Please upgrade to the latest version. And you could add trivy registry configuration refer to this PR
https://github.com/goharbor/harbor/pull/21193
